scene/gui/code_edit.h

```diff
@@ -332,6 +334,9 @@ class CodeEdit : public TextEdit {
 	void set_auto_indent_enabled(bool p_enabled);
 	bool is_auto_indent_enabled() const;
 
+	void set_tab_always_indents(const bool p_tab_always_indents);
```

Comment:
```suggestion
	void set_tab_always_indents_enabled(bool p_enabled);
```
No `const` on simple types, same about wording as below